CHANGELOG.md

```diff
@@ -2,6 +2,21 @@
 All notable changes to this project will be documented in this file.
 This project adheres to [Semantic Versioning](http://semver.org/).
 
+## Unreleased
+### Added
+- `scriptworker.artifacts` is a new submodule that defines artifact behavior
+- we now support `pushapk` scriptworker instance types in `cot.verify`
+
+### Changed
+- `freeze_values` is now `get_frozen_copy`, and now returns a frozen copy instead of modifying the object in place.
+- `unfreeze_values` is now `get_unfrozen_copy`
+- `check_config` now calls `get_frozen_copy` on the `config` before comparing against `DEFAULT_CONFIG`
+- `create_config` calls `get_unfrozen_copy`, resulting in a recursively frozen config
+- `DEFAULT_CONFIG` now uses `frozendict`s and `tuple`s in nested config items.
+- `.asc` files are now forced to `text/plain`
+- `text/plain` artifacts are now gzipped
```

Comment:
Bikeshed: Not sure it's worth being called out, but json, html and xml files are also gzipped https://github.com/mozilla-releng/scriptworker/pull/53/files#diff-ecbeaeb0c758367cccb8ff577090c03fR25